tests/ibc-hooks/ibc_middleware_test.go

```diff
@@ -573,8 +573,8 @@ func (suite *HooksTestSuite) SetupCrosschainSwaps(chainName Chain, withAckTracki
 	suite.SetupPools(chainName, []sdk.Dec{sdk.NewDec(20), sdk.NewDec(20)})
 
 	// Setup contract
-	chain.StoreContractCode(&suite.Suite, "./bytecode/swaprouter.wasm")
-	swaprouterAddr := chain.InstantiateContract(&suite.Suite,
+	chain.StoreContractCode(&suite.Suite, "./bytecode/poolmanager.wasm")
```

Comment:
This is a swaprouter contract which should stay swaprouter

```suggestion
	chain.StoreContractCode(&suite.Suite, "./bytecode/swaprouter.wasm")
```